src/modules/PublicLab.MainImageModule.js

```diff
@@ -25,10 +25,10 @@ module.exports = PublicLab.MainImageModule = PublicLab.Module.extend({
         _module.image.onload = function() {
           var heightDropdown = this.height;
           var widthDropdown = this.width;
-          if (this.width > 340) {
-            var aspectRatio = this.width / 340;
-            widthDropdown = 340;
-            heightDropdown = this.height / aspectRatio;
+          if (this.height > 180) {
```

Comment:
Can you also give screenshot in desktop view and verify that after uploading, the CSS is not changed.

Reply:
![image](https://user-images.githubusercontent.com/47106543/93220369-0a628e00-f78a-11ea-8682-a750ccce825f.png)
This is the screenshot in desktop full screen

Reply:
![image](https://user-images.githubusercontent.com/47106543/93220491-3ed64a00-f78a-11ea-9984-1b3af72b51a7.png)
So vertical screenshot are looking very small since I fixed the height instead of width but if we let it overflow it will start overlapping again.

Reply:
@jywarren I believe it's not related to this issue. An additional class `row` inclusion here
https://github.com/publiclab/plots2/blob/7984fc91c982f8ff1f4aa0b322ebecdb010382a4/app/views/editor/rich.html.erb#L84

will fix that. It's a fix needed on plots2 implementation since it displays fine on example html in this repo.

Reply:
Hey @daemon1024, 
Sorry for late reply here! Yes you are correct it is something which was not mentioned in the original issue description 👍 
Great work so far 🎉 🎉 🎉